src/app/pages/ArticlePage/index.stories.tsx

```diff
@@ -230,3 +236,25 @@ export const ArticlePageWithElectionBanner = {
     chromatic: { disableSnapshot: true },
   },
 };
+
+export const ArticlePageWithLiteSiteCTA = (
```

Comment:
Let's disable snapshots, as this is already captured by `TestArticlePageWithLiteSiteCTA`

Reply:
cc @Isabella-Mitchell can we do this before we merge, because the UI Tests snapshots are a bit messed up at the moment